pkg/execution/state/state.go

```diff
@@ -65,6 +65,9 @@ type Identifier struct {
 	WorkspaceID uuid.UUID `json:"wsID"`
 	// If this is a rerun, the original run ID is stored here.
 	OriginalRunID *ulid.ULID `json:"oRunID,omitempty"`
+	// PriorityFactor is the overall priority factor for this particular function
+	// run.  This allows individual runs to take precedence within the same queue.
+	PriorityFactor *int64 `json:"pf,omitempty"`
```

Comment:
Better to provide a note what the `int64` is for.

e.g. The higher the number, the higher the priority within the range.

Probably don't need an essay here describing it as the seconds, but good to have a little note here so for people reading the code, we know this number actually represent seconds.